frontend/packages/client/.env.development

```diff
@@ -0,0 +1 @@
+GENERATE_SOURCEMAP=false
```

Comment:
this is needed to avoid ongoing bug: https://github.com/facebook/create-react-app/pull/11752